Aggregator.Core.Plugin/Aggregator.Core.Plugin.csproj

```diff
@@ -0,0 +1,201 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug-2018</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{FEA76835-61B3-4FC7-9D6F-50D39A7A8258}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Aggregator.Core</RootNamespace>
+    <AssemblyName>TFSAggregator2.Core</AssemblyName>
+    <TargetFramework Condition="'$(TargetFramework)' == ''">net461</TargetFramework>
+    <FileAlignment>512</FileAlignment>
+    <Configurations>Debug-2013;Release-2013;Debug-2015;Release-2015;Debug-2015.1;Release-2015.1;Debug-2015.2;Release-2015.2;Debug-2017;Release-2017;Debug-2017.2;Release-2017.2;Debug-2018;Release-2018</Configurations>
```

Comment:
Ideally, there shouldn't be any `Debug-XYZ` / `Release-XYZ` configurations left if all SDK projects inherit a default `TfsVersion` parameter from a `Directory.Build.props` file (auto-imported). The properties could then be conditioned on the `TfsVersion` parameter instead of the configuration (or a set in the Directory.Build.props file).

This would allow adding/removing tfs versions without fiddling with solution and projects configurations.

Reply:
I tried that, But Visual Studio doesn't pick that up and it breaks the compiler directives in the IDE. But it is a nice next step. For now I'm glad it actually builds. I also tried auto-importing TargetFramework, but that triggers an Upgrade Wizard in VS.

Reply:
I tried provisioning the <configurations> property using a filter expression using the TfsVersions, but the IDE doesn't like that either, it auto-generates all the project configs on the fly.

Reply:
Are the configurations still in the solution? I'd suggest starting out with a completely new solution file. Or leave it at the status quo.

Reply:
I think for now I'll leave it. I'm working on removing as much as possible from the configurations, to make it as easy as I can to maintain them. copy-pasting 2 sections into 6 files every 3 months is doable.